Salary App/ViewController.swift

```diff
@@ -66,33 +70,33 @@ class ViewController: UIViewController, UITextFieldDelegate {
             let salary = Int(self.salaryTextField.text!) ?? 0
             monthlySalary =  salary / 12
```

Comment:
Could we seperate out this calculation logic into some kinda of calculator object, pass the function an enum and based on that case do some calculations based on the salary you pass in.

Also it would allow you to write unit tests on it easily which is something you should look into next